### ISSUE TITLE
Suggestion: feeding received data to other libraries such as expat

### ISSUE THREAD

**Issue by gevaerts** [bagder/libcurl-video-tutorials#4]
The expat xml library (https://github.com/libexpat/libexpat) wants its data in a way that is well matched with the curl API. This (or some other similar library) may be worth showing.

The libcurl "xmlstream" example does exactly this.

**Comment by bagder** [comment]
Agreed. Shows real-world use of the write callback other than just saving it like in the "receive data" episode.

The xmlstream example: https://curl.haxx.se/libcurl/c/xmlstream.html
